sequel/lib/criteriaProcessor.js

```diff
@@ -691,6 +691,11 @@ CriteriaProcessor.prototype.prepareCriterion = function prepareCriterion(key, va
       }
 
       break;
+
+    default:
+      var err = new Error('Unknown filtering operator: "' + key + "\". Should be 'startsWith', '>', 'contains' or similar");
+      err.operator = key;
+      throw err;
```

Comment:
I should note that every function call is already wrapped in a try/catch, for example in sails-postgresql

```
            try {
              _query = sequel.find(table, instructions);
            } catch(e) {
              return next(e);
            }
```